Core/HLE/sceKernelHeap.cpp

```diff
@@ -110,6 +126,21 @@ static u32 SysMemForKernel_536AD5E1()
 	return 0;
 }
 
+//should have option variable
```

Comment:
hm, why not declare the extra option parameter then? even if it's not used and only logged.